cmd/edit.go

```diff
@@ -0,0 +1,117 @@
+		Long:  editDashboardsLongDescription,
+		Args:  cobra.NoArgs,
+		RunE:  editDashboardsCmd,
+	}
+	editDashboardsCmd.Flags().StringSliceP(cobraext.DashboardIDsFlagName, "d", nil, cobraext.DashboardIDsFlagDescription)
+	editDashboardsCmd.Flags().Bool(cobraext.TLSSkipVerifyFlagName, false, cobraext.TLSSkipVerifyFlagDescription)
+	editDashboardsCmd.Flags().Bool(cobraext.AllowSnapshotFlagName, false, cobraext.AllowSnapshotDescription)
+
+	cmd := &cobra.Command{
+		Use:   "edit",
+		Short: "Edit package assets",
+		Long:  editLongDescription,
+	}
+	cmd.AddCommand(editDashboardsCmd)
+	cmd.PersistentFlags().StringP(cobraext.ProfileFlagName, "p", "", fmt.Sprintf(cobraext.ProfileFlagDescription, install.ProfileNameEnvVar))
+
+	return cobraext.NewCommand(cmd, cobraext.ContextPackage)
+}
+
+func editDashboardsCmd(cmd *cobra.Command, args []string) error {
+	cmd.Println("Make Kibana dashboards editable")
+
+	dashboardIDs, err := cmd.Flags().GetStringSlice(cobraext.DashboardIDsFlagName)
+	if err != nil {
+		return cobraext.FlagParsingError(err, cobraext.DashboardIDsFlagName)
+	}
+
+	common.TrimStringSlice(dashboardIDs)
+
+	var opts []kibana.ClientOption
+	tlsSkipVerify, _ := cmd.Flags().GetBool(cobraext.TLSSkipVerifyFlagName)
+	if tlsSkipVerify {
+		opts = append(opts, kibana.TLSSkipVerify())
+	}
+
+	allowSnapshot, _ := cmd.Flags().GetBool(cobraext.AllowSnapshotFlagName)
+	if err != nil {
+		return cobraext.FlagParsingError(err, cobraext.AllowSnapshotFlagName)
+	}
+
+	profile, err := cobraext.GetProfileFlag(cmd)
+	if err != nil {
+		return err
+	}
+
+	kibanaClient, err := stack.NewKibanaClientFromProfile(profile, opts...)
+	if err != nil {
+		return fmt.Errorf("can't create Kibana client: %w", err)
+	}
+
+	kibanaVersion, err := kibanaClient.Version()
+	if err != nil {
+		return fmt.Errorf("can't get Kibana status information: %w", err)
+	}
+
+	if kibanaVersion.IsSnapshot() {
+		message := fmt.Sprintf("editing dashboards from a SNAPSHOT version of Kibana (%s) is discouraged. It could lead to invalid dashboards (for example if they use features that are reverted or modified before the final release)", kibanaVersion.Version())
+		if !allowSnapshot {
+			return fmt.Errorf("%s. --%s flag can be used to ignore this error", message, cobraext.AllowSnapshotFlagName)
+		}
+		fmt.Printf("Warning: %s\n", message)
+	}
+
+	if len(dashboardIDs) == 0 {
+		dashboardIDs, err = promptDashboardIDs(kibanaClient)
+		if err != nil {
+			return fmt.Errorf("prompt for dashboard selection failed: %w", err)
+		}
+
+		if len(dashboardIDs) == 0 {
+			fmt.Println("No dashboards were found in Kibana.")
+			return nil
+		}
+	}
+
+	kibanaHost := kibana.GetAddress(*kibanaClient)
+	urls := ""
+	for _, dashboardID := range dashboardIDs {
+		err = kibanaClient.SetManagedSavedObject("dashboard", dashboardID, false)
+		if err != nil {
+			return fmt.Errorf("failed to make dashboards editable: %w", err)
+		}
+		urls += fmt.Sprintf("\n%s/app/dashboards#/view/%s", kibanaHost, dashboardID)
+	}
```

Comment:
Nit. Some Go-style ideas here:

* We can use `url` methods to work with URLs in a safer way, we would need to parse the address as url.
```suggestion
	kibanaHost := kibana.GetAddress(*kibanaClient)
	kibanaURL, err := url.Parse(kibanaHost)
	if err != nil {
	   ...
	}
	dashboardsURL, err := kibanaURL.JoinPath("app", "dashboards")
	if err != nil {
	   ...
	}
	urls := ""
	for _, dashboardID := range dashboardIDs {
		err = kibanaClient.SetManagedSavedObject("dashboard", dashboardID, false)
		if err != nil {
			return fmt.Errorf("failed to make dashboards editable: %w", err)
		}
		dashboardURL = *kibanaURL
		dashboardURL.Fragment = "/view/" + dashboardID
		urls += "\n" + dashboardURL.String()
	}
```

* It is usually better to use a string builder, than to concatenate strings. It is probably fine here as there won't be a lot of text.
```suggestion
	kibanaHost := kibana.GetAddress(*kibanaClient)
	var urls strings.Builder
	for _, dashboardID := range dashboardIDs {
		err = kibanaClient.SetManagedSavedObject("dashboard", dashboardID, false)
		if err != nil {
			return fmt.Errorf("failed to make dashboards editable: %w", err)
		}
		fmt.Fprintf(&urls, "\n%s/app/dashboards#/view/%s", kibanaHost, dashboardID)
	}
	...
	cmd.Println(fmt.Sprintf("Dashboards URLs:%s", urls.String()))
```
* Or we could also make `urls` an array of strings `[]string`, append them here and use `strings.Join` later.
* Consider splitting this loop in two, one to do the operation, and another one to print the urls. This could also evolve to different functions.